server/athenian/api/auth.py

```diff
@@ -19,10 +19,11 @@ def __init__(self, error, status_code):
 class User:
     """User profile information."""
 
-    def __init__(self, profile: str, name: str, picture: str, updated_at: str, **kwargs):
+    def __init__(self, sub: str, email: str, name: str, picture: str, updated_at: str, **kwargs):
```

Comment:
I'm a newbie with Python, but... could this change cause https://athenianco.atlassian.net/browse/ENG-138